data-structures-stacks/count-values.js

```diff
@@ -0,0 +1,13 @@
+/* exported countValues */
+
+function countValues(stack) {
+  if (stack.print() === 'Stack { <empty> }') {
```

Comment:
as noted in the instructions, you don't want to use the `print` method in the solutions.